wordpress-mobile/WordPress-iOS
# Prepublishing Nudges

Merge the PPN to `develop`. :)

### To test

1. Create a new post
2. Tap Publish
3. Check if the Nudges view appears

PR submission checklist:
